Baseline Ranked Variables- High Priority.Rmd

```diff
@@ -0,0 +1,5324 @@
+
+
+dtPREWORKRES_summary <- PREWORKRES %>% group_by(meds) %>%  dplyr::summarize(n=n(), percentage=100*n/nrow(.))  %>% dplyr::ungroup() %>%  dplyr::select(meds, n, percentage)
+
+
+dtPREWORKRES_summary %>%  gt(rowname_col = "row_lab") %>%  
+  fmt_number(columns = "percentage", decimals = 2) %>% 
+  tab_header(title = md("PREWORKRES: Daily Responsibilities of Longest Held Job Include Driving")) %>% 
+  cols_label(n= md("**Number of Participants**"), meds = md("**Answer**"), percentage = md("**Percentage of Participants**")) %>% 
+  # summary_rows(
+  #   column = c(n,percentage),
+  #   fns= list(
+  #     Sum= ~sum(.)
+  #   ), decimals=0) %>%
+  grand_summary_rows(columns=c(n, percentage),fns = ~sum(.,na.rm = T))|>
+  tab_options(
+    stub.font.weight = "bold"
+  ) %>%
+  tab_style(
+    style = list(
+      cell_text(weight = "bold")
+    ),
+    locations = cells_body(
+      columns = meds,
+    )
+  ) %>% opt_row_striping()
+
+```
+
+
+
+
+
+
+
+
+
+
+
+
+
+
+## Map of Current Addresses
+```{r, echo=FALSE, warning=FALSE, message=FALSE}
+
+library(ggmap)
+library(maps)
+library(mapdata)
+library(sf)
+library(zipcodeR)
+library(viridis)
+library(ggthemes)
+#library(albersusa)
+
+states <- map_data("state")
+us_base <- ggplot(data = states) +
+  geom_polygon(aes(x = long, y = lat, fill = region, group = group), color = "white") +
+  coord_fixed(1.3) +
+  guides(fill=FALSE)  # do this to leave off the color legend
+
+
+ca_base <- ggplot(data = states, mapping = aes(x = long, y = lat, group = group)) + 
+  coord_fixed(1.3) + 
+  geom_polygon(color = "black", fill = "gray")
+#ca_base + theme_nothing()
+
+
+# Example with zip codes : https://stackoverflow.com/questions/59801437/is-there-an-r-package-that-can-take-zipcodes-and-create-a-heat-map-of-the-united
+
+
+#Attempt 1
+ zipc_heat <- data_tib_all4 %>% mutate(ZIP= str_sub(data_tib_all4$D_121490150_D_202784871,1,5),
+                                       State= D_121490150_D_195068098)
+
+ zipc_heat <- zipc_heat %>%  select(ZIP) %>%  filter(!is.na(ZIP) & ZIP!="Gfchh" & str_sub(ZIP,1,1)!=" " & str_sub(ZIP,1,2)!="  " & str_sub(ZIP,5,5)!=" " & str_sub(ZIP,4,5)!="  ") ## need to remove the 4 digit zip code and random letter zip code
+
+ #zipc_heat$ZIP <- as.numeric(zipc_heat$ZIP)
+
+ ZP_HT <- as.data.frame(table(zipc_heat))
+
+
+ #ZP_HT$ZIP<- clean.zipcodes(ZP_HT$ZIP)
+ ZP_HT_US<-aggregate(data.frame(count=ZP_HT$Freq),list(zip=ZP_HT$ZIP),length)
+
+
+ 
+# EXAMPLE OF PER COUNTY GRAPH OF JUST CA- have to figure out how to add counties of all countries and populations equal to our participant population 
+   # https://eriqande.github.io/rep-res-web/lectures/making-maps-with-R.html
+  #https://jtr13.github.io/cc19/different-ways-of-plotting-u-s-map-in-r.html
+ #https://www.sharpsightlabs.com/blog/how-to-create-a-crime-heatmap-in-r/
+ #https://rpubs.com/jcraggy/841199
+ 
+ 
+ #Attemp1 10 
+ zip_to_lat_lon_North.America <- read.csv("C:/Users/dowlingk2/Downloads/zip_to_lat_lon_North America.csv")
```

Comment:
This is reading a file on your local C drive. If you need. toinclude this file it needs to be in. the repo.